docs/pages/versions/unversioned/sdk/application.md

```diff
@@ -0,0 +1,130 @@
+- [`Application.getInstallReferrerAsync()`](#applicationgetinstallreferrerasync) (Android only)
+- [`Application.getFirstInstallTimeAsync()`](#applicationgetfirstinstalltimeasync)
+- [`Application.getLastUpdateTimeAsync()`](#applicationgetlastupdatetimeasync) (Android only)
+
+## Constants
+
+### `Application.applicationName`
+
+The human-readable name of the application.
+
+- e.g., `Expo`, `Yelp`, `Instagram`
+
+### `Application.bundleId`
+
+The bundle ID of the application.
+
+- e.g., `com.cocoacasts.scribbles`, `com.apple.Pages`
+
+### `Application.nativeAppVersion`
+
+The `Info.plist` value for `CFBundleShortVersionString` on iOS and the version name set by `version` in app.json on Android at the time the native app was built.
+
+- e.g., `2.11.0`
+
+### `Application.nativeBuildVersion`
+
+The `Info.plist` value for `CFBundleVersion` on iOS (set with `ios.buildNumber` value in `app.json` in a standalone app) and the version code set by `android.versionCode` in app.json on Android at the time the native app was built.
+
+- e.g., `2.11.0`
+
+### `Application.androidId`
+
+**Android only.** A hexademical `string` unique to each combination of app-signing key, user, and device. The value may change if a factory reset is performed on the device or if an APK signing key changes. For more information about how the platform handles ANDROID_ID in Android 8.0 (API level 26) and higher, see [Android 8.0 Behavior Changes.](https://developer.android.com/about/versions/oreo/android-8.0-changes.html#privacy-all)
+
+Note: In versions of the platform lower than Android 8.0 (API level 26), this value remains constant for the lifetime of the user's device.
+
+See: https://developer.android.com/reference/android/provider/Settings.Secure.html#ANDROID_ID
+
+- e.g., `"dd96dec43fb81c97"`
+
+## Methods
+
+### `Application.getIosIdForVendorAsync()`
+
+**iOS only.** Gets the iOS identifier for vender [(IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) value, an alphanumeric string that uniquely identifies a device to the app’s vendor.
+
+Since this is the IDFV value, its value can change if all apps from the current apps vendor have been uninstalled.
+
+#### Returns
+
+A `Promise` that resolves to a `string` of the IDFV for the app.
+
+**Examples**
+
+```js
+await Application.getIosIdForVendorAsync();
+//"FCDBD8EF-62FC-4ECB-B2F5-92C9E79AC7F9"
+```
+
+### `Application.getInstallReferrerAsync()`
+
+**Android only.** Gets the referrer URL of the installed app with [`Install Referrer API`](https://developer.android.com/google/play/installreferrer) from Google Play Store.
+
+#### Returns
+
+A `Promise` that resolves to a `string` of the referrer URL of the installed app.
+
+**Examples**
+
+```js
+await Application.getInstallReferrerAsync();
+//"utm_source=google-play&utm_medium=organic"
+```
+
+### `Application.getFirstInstallTimeAsync()`
+
+Gets the first time the app was installed onto the device.
```

Comment:
```suggestion
Gets the time the app was first ever installed onto the device.
```

(is this accurate? and how does it work with multiple user accounts?) What about web?